packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -893,9 +928,9 @@ <h2>Ranges Vertical</h2>
           thumb.classList.add("iui-active");
           sliderContainer.classList.add("iui-grabbing");
           var ownerDoc = thumb.ownerDocument;
-          var track = sliderContainer.querySelector(".iui-slider-rail");
+          var track = sliderContainer.querySelector(".iui-slider::before");
```

Comment:
This can be removed since `track` is no longer used in the code.

Reply:
Done.